### ISSUE TITLE
monkey-patch gets overwritten after every VS Code restart and customize-ui doesn't work as a result

### ISSUE THREAD

**Issue by NwraliMedeu** [iocave/customize-ui#39]
Please help me with the issue stated in the title.

**Comment by knopp** [comment]
What platform?

**Comment by midanosi** [comment]
> What platform?

I don't know what the cause is, but I've been having the same issue of customize-ui simply not working for the last couple days.
Platform that it isn't working on for me is MacOS.
The OP also posted this exact same issue on the Monkey Patch github page, and someone replied there stating platform macos too.

Everything seems to work fine on Windows for me still.

**Comment by Nurali9999** [comment]
> What platform?

Sorry, my platform is Linux Mint 19.3 64-bit.

**Comment by knopp** [comment]
@midanosi, do you have VSCode installed in Applications folder or are you running it from downloads?

@Nurali9999 are you by any chance running VSCode from some container (i.e. Snap)?

**Comment by Nurali9999** [comment]
@knopp If my VSCode were in a container, would it be in my home directory, or a different directory?

**Comment by Nurali9999** [comment]
@knopp I am sorry for not being clear, I meant "How can I check if my VSCode is in a container?"

**Comment by midanosi** [comment]
> @midanosi, do you have VSCode installed in Applications folder or are you running it from downloads?

Just checked, running from Applications folder. Is there a set of debugging steps I could run to help you acquire logging info or the like?

**Comment by midanosi** [comment]
So, I managed to fix my problem by attempting to modify a 'Customize UI' setting via the Settings UI, which triggered the notification describing Monkey Patch wasn't enabled (I hadn't received a notification that prompting me to enable it before this), and I followed the notification to restart, and it started working again.

Thanks again for making this iocave, it really makes my DX with VSCode so so much better.
Sorry for the PEBCAK, although maybe there is a problem where the notification to tell you to enable monkey patch is not appearing?

**Comment by twome** [comment]
@midanosi Thank you for the update! Exiting the Settings UI fixed the same issue for me.

**Comment by NwraliMedeu** [comment]
Sorry for abandoning this issue. I no longer use the extension. Thank you for trying to fix my past problem.

**Comment by brendan8c** [comment]
VS Code worked well for me.
I installed the Customize UI plugin.
This plugin prompted me to reload my VS Code.
After rebooting VS Code stopped opening.
  VS Code hangs in open processes in the task manager, but the program window itself does not start!

I restarted my computer, but that didn't help me launch my VS Code.
I uninstalled VS Code and reinstalled it.
After installation, VS Code starts, plugins start, a window pops up from the Customize UI plugin asking me to restart my VS Code, I restart and VS Code stops working again!

Your plugin is conflicting with something. It doesn't work correctly.
![23_13_13 30-12-2022](https://user-images.githubusercontent.com/61526280/210109580-e02b20ae-d14d-4a89-b57d-e9f9a3b41a0a.png)
![23_20_08 30-12-2022](https://user-images.githubusercontent.com/61526280/210109592-837a4e0f-08d3-4856-a009-0677309cdb1d.png)
My settings.json for vscode:
[settings.json.txt](https://github.com/iocave/customize-ui/files/10326190/settings.json.txt)
